doc/conf.py

```diff
@@ -119,23 +119,24 @@
 
 # -- Options for HTML output ----------------------------------------------
 
-on_rtd = os.environ.get('READTHEDOCS', None) == 'True'
+on_rtd = os.environ.get("READTHEDOCS", None) == "True"
 
 if not on_rtd:
     # only import and set the theme if we're building docs locally
     import sphinx_rtd_theme
-    html_theme = 'sphinx_rtd_theme'
+
+    html_theme = "sphinx_rtd_theme"
```

Comment:
Since Sphinx seems to use magic variables, I am unsure what pattern to use to suppress this warning. Might be worthwhile just manually dismissing this whenever we make an edit to this file. That way we'd still get CodeQL analysis everywhere else (where we are unlikely to use such weird patterns).